person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    true if of_age? || @parent_permission
```

Comment:
[OPTIONAL] Comparisons by default return true or false, so I think we don't need to specify to return `true if...`.

Reply:
done